psalm-baseline.xml

```diff
@@ -8,15 +8,22 @@
       <code>$array</code>
     </MixedAssignment>
   </file>
+  <file src="src/Persistence/SqlAllMappingsLookup.php">
```

Comment:
[`Wikimedia\Rdbms\JoinGroupBase::join()`](https://doc.wikimedia.org/mediawiki-core/REL1_43/php/classWikimedia_1_1Rdbms_1_1JoinGroupBase.html#a0c0d3a7c178e9df6e5754025ae79bdb1) is marked as internal and I don't know how to replace it